java/common/src/main/java/org/ray/util/LambdaUtils.java

```diff
@@ -0,0 +1,58 @@
+package org.ray.util;
+
+import java.io.Serializable;
+import java.lang.invoke.SerializedLambda;
+import java.lang.reflect.Method;
+import java.util.WeakHashMap;
+
+/**
+ * see http://cr.openjdk.java.net/~briangoetz/lambda/lambda-translation.html
+ */
+public final class LambdaUtils {
```

Comment:
There're 2 layers of cache here.
RayFunc -> SerializedLambda
SerializedLambda -> MethodId.
Should we merge them?

Reply:
yes, i will do that